### PR TITLE
Ds005/competencies per band

### PR DESCRIPTION
**Pull request by clarebuckley**
These changes will also remove console errors currently in dev

### CODE REVIEW

**Review by aowenmeehanwork — APPROVED** [review]
LGTM

File: src/app/table-page/table-page.component.html [hunk 1]
```diff
@@ -15,4 +15,4 @@
 </ag-grid-angular>
 </div>
 
-<button (click)="navigateToDetailView('role', 1)">hi</button>
+<button (click)="navigateToDetailView('band', 1)">Test button</button>
```
Comment (micalexis) [inline src/app/table-page/table-page.component.html]: what is the button for? why is it called test button

**Review by micalexis — COMMENTED** [review]
`Ds` stories should be merged into their relevant `US` stories and merge the that into `Dev` not directly!

File: src/app/table-page/table-page.component.html (hunk 1 above)
Comment (clarebuckley) [reply src/app/table-page/table-page.component.html]: This was added as a placeholder to test functionality while other team members worked on clickable cells in the table, I'll remove this from this pull request

**Review by micalexis — COMMENTED** [review]
👍

**Comment by clarebuckley** [comment]
Errors in console were due to a missing npm package on my local machine, no changes have been made to code since approval from PO

**Pull request merged**